Get-NetView.psm1

```diff
@@ -2932,7 +2956,7 @@ function Get-NetView {
 
         throw $_
     } finally {
-        Completion -Src $workDir
+        Completion -Src $workDir -SkipWindowsRegistry $SkipWindowsRegistry
```

Comment:
Can you elaborate more on why this is needed?  Any files contained within base Get-NetView execution would automatically be contained in the generated ZIP.

Reply:
The reg export files are large, so even in the unzipped folder, it was better to keep them zipped to prevent a large folder. The exported registry files are zipped separately from the rest of the NetView contents in Completion because it is guaranteed at this point that all reg export commands have completed.